common/model-views.cpp

```diff
@@ -5815,7 +5824,8 @@ namespace rs2
                                 if (selected < static_cast<int>(labels.size() - files_labels.size()))
                                 {
                                     //Known preset was chosen
-                                    auto new_val = opt_model.range.min + opt_model.range.step * selected;
+                                    auto new_val
+                                        = opt_model.range.min + opt_model.range.step * counters[selected];
```

Comment:
Not sure I agree that there should be a mult here by `step` -- but I see you haven't changed this code